### ISSUE TITLE
Duration and curve required on next page method

### ISSUE THREAD

**Issue by ElZombieIsra** [serenader2014/flutter_carousel_slider#134]
You should mark as required the params `duration` and `curve` in the `nextPage` method inside the controller or give them some default values because the plugin will not change the page 
unless these params are given and not null

This is the code where the plugin fails 
```
/// Animates the controlled [CarouselSlider] to the next page.
///
/// The animation lasts for the given duration and follows the given curve.
/// The returned [Future] resolves when the animation completes.
Future<void> nextPage({Duration duration, Curve curve}) {
  return _state.pageController.nextPage(duration: duration, curve: curve);
}
```

**Comment by serenader2014** [comment]
You're right. I'm adding default value to the `nextPage` and `previousPage` and `animateToPage` method in the `feature/v2` branch. Will merge to the master branch when I'm finishing this branch.

**Comment by serenader2014** [comment]
Fixed in v2.1.0.
